scripts/run_katana

```diff
@@ -1,60 +1,62 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-from datetime import datetime
-import sys
-import os
 import argparse
+import os
+import sys
+from datetime import datetime
+
+import dateparser
 import numpy as np
-import pandas as pd
 
 from katana.framework import Katana
 
+
 def run():
     '''
     katana is a command line program designed to take a config file. This
     will be fed to the run_katana function within framework.
     '''
-    
+
     # Parse arguments
     p = argparse.ArgumentParser(description='Run Katana wind system.')
 
-    p.add_argument('-sd','--start_date', required=True,
-                    help="start date for run")
+    p.add_argument('-sd', '--start_date', required=True,
```

Comment:
Right now, I believe you have to pass the date as a string. So it would be `--start_date='2010-10-1 00:00` and it will fail if it's not like that. The only change here was linting so it will operate the same as it has been in the current version.